src/main.c

```diff
@@ -19,6 +19,10 @@
 
 int main(int argc, char **argv)
 {
+#ifdef __sparc
+    EnableSparcMisalignEmulation();
```

Comment:
Would like to see this be more generic, like a single call out to something like `UtilCpuSpecificInit()` where inside that (in util-cpu.c) you would have this ifdef'd call to `EnableSparcMisalignEmulation`